modules/virt-deleting-dvs.adoc

```diff
@@ -5,9 +5,19 @@
 :_content-type: PROCEDURE
 [id="virt-deleting-dvs_{context}"]
 
-= Deleting a data volume
+= Deleting data volumes
 
-You can delete a data volume by using the `oc` command-line interface (CLI).
+You can delete a data volume if needed by using the `oc` command-line interface (CLI), but be aware that doing so also deletes the underlying provisioned volume claim (PVC).
+
+[NOTE]
+====
+When you delete a virtual machine, the data volume it uses is automatically deleted.
```

Comment:
When you delete a virtual machine, the data volume it uses is automatically deleted. - >  When deleting a virtual machine which includes a dataVolumeTemplate section referencing a data volume, that DV will be deleted as well as the PVC which is linked to it. A stand alone data volume, used by a VM will not be deleted when the VM is deleted.